sysdump/constants.go

```diff
@@ -53,6 +53,7 @@ const (
 	eniconfigsFileName                       = "aws-eniconfigs-<ts>.yaml"
 	gopsFileName                             = "gops-%s-%s-<ts>-%s.txt"
 	hubbleDaemonsetFileName                  = "hubble-daemonset-<ts>.yaml"
+	hubbleFlowsFileName                      = "hubble-flows-%s-<ts>.txt"
```

Comment:
Should the filename be `.json` because it will contain JSON?

Reply:
yeah let's do it 👍 , updating...

Reply:
done!